docs/swagger.yaml

```diff
@@ -39,6 +41,10 @@ paths:
         - $ref: '#/components/parameters/Authorization'
         - $ref: '#/components/parameters/Ocp-Apim-Subscription-Key'
         - $ref: '#/components/parameters/Merchant-Serial-Number'
+        - $ref: '#/components/parameters/Vipps-System-Name'
```

Comment:
What purpose does these serve?

Reply:
@matsjohans  thought they should be included since they are standard vipps headers. I guess he would like to use them in splunk for debugging or something like that.

Reply:
Mats says we should add the optional headers now, and we should add logic for these at some point.